Atos20/Crate
# Display style survey

PR Template:
What’s this PR do?  

-  it displays the `<StyleSurvey/>` component
- it adds a new path to the user's paths
- it redirects the user to `/style-preferences` path when the user clicks on the`+Subscribe` button
- it styles  the StyleSurvey component
- it reuses UI component to create the Style survey
- It adds a new case to the user reducer
- it adds a new property to the store state to keep track of the user has completed the survey
- it adds a pull request to the project repository

Where should the reviewer start?  

-  cd into the `Crate/code/web/src` and open 
    - /modules/StyleSurvey/StyleSurvey.js
    - /modules/crate/Item.js 
    - /modules/crate/List.js
    - /modules/user/api/actions.js
    - /setup/store.js

How should this be manually tested?  
- cd into `code/web` and run `npm start` to start the server on `localhost:3000` 
- cd into `code/api` and run `npm start` to start the server on `localhost:8000` \
- make sure the files have finished compiling
- log in as a user
- navigate to `/crates`
- select a subscription

Any background context you want to provide?  
 - As a team, we have been assigned with creating a style survey that a user can fill out and find out the unique style.
 - Our team is conformed of 4BE students that will be modifying the existing schemas built in the project
 
What are the relevant tickets?  
 - closes  #19 
 - closes #9 
 - closes #17 
 - closes #20 
 - closes #21 

Screenshots (if appropriate)  

 ![Screen Shot 2020-12-07 at 10 37 38 PM](https://user-images.githubusercontent.com/56229864/101444497-d6a48980-38dc-11eb-87c2-aa894fef9a39.png)
